framework/levure.livecodescript

```diff
@@ -1888,8 +1900,8 @@ private command buildExtension pFilename, pOutputFolder
 
   # Look in folder of pFilename as well as 'lci' subdirectory of pFilename folder.
```

Comment:
@montegoulding Please update the comment to use correct variable name

Reply:
done